src/test/java/uk/gov/hmcts/reform/blobrouter/reconciliation/service/datetimechecker/SummaryCronRelevancyCheckerTest.java

```diff
@@ -0,0 +1,62 @@
+package uk.gov.hmcts.reform.blobrouter.reconciliation.service.datetimechecker;
+
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.time.ZonedDateTime;
+import java.util.stream.Stream;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static uk.gov.hmcts.reform.blobrouter.util.TimeZones.EUROPE_LONDON_ZONE_ID;
+
+class SummaryCronRelevancyCheckerTest {
+
+    @ParameterizedTest(name = "{arguments}")
+    @MethodSource("provideCheckerTestScenarios")
+    void should_test_checker_under_provided_scenario(CheckerTestScenario scenario) {
+        SummaryReportCronRelevancyChecker checker = new SummaryReportCronRelevancyChecker(scenario.cron);
+
+        ZonedDateTime timeUnderTest = currentTimeWithHourShiftedTo(scenario.hourUnderTest);
+        boolean actualResult = checker.isTimeRelevant(timeUnderTest);
+
+        assertEquals(scenario.expectedResult, actualResult, scenario.errorMessage);
+    }
+
+    private static Stream<Arguments> provideCheckerTestScenarios() {
+        return Stream.of(
+            Arguments.of(new CheckerTestScenario("0 */30 1-6 * * *", true, 5, "The next cron would fire at 6am so 5am is relevant")),
+            Arguments.of(new CheckerTestScenario("0 */30 6 * * *", false, 7, "The next cron would fire next day at 1am so 7am is irrelevant")),
+            Arguments.of(new CheckerTestScenario("0 */30 3 * * *", false, 3, "The next cron would fire next day at 1am so 3am is irrelevant")),
```

Comment:
does this cron expression not include 3:30?

Reply:
Unfortunately, it does, but I spoke with Mustafa and he sort of mentioned changing the cron to hourly, so I didn't want to make the implementation more complex, if it's not going to be used.

Reply:
But answering the question - I probably need to revisit the error messages and add tests which shows the minutes are ignored!